src/main/resources/application-local.properties

```diff
@@ -0,0 +1,15 @@
+#spring
+spring.main.banner-mode=console
+
+#spring batch
+spring.datasource.driverClassName=org.postgresql.Driver
+spring.datasource.url=<POSTGRES-URL-GOES-HERE>
```

Comment:
How is this replaced?

Reply:
This properties file is to be used locally and so the values are replaced by the user manually, according to how the environment was configured (datasource running locally or on cloud? I-on core should be running locally, but on what address/port?)